docs/home/DEPRECATION-NOTE.md

```diff
@@ -32,3 +32,15 @@ The functionality that the Logstash is serving in Epiphany - exporting csv can r
 
 Removed: Epiphany 2.0
 Issue: [2833](https://github.com/epiphany-platform/epiphany/issues/2833)
+
+## Next steps
+
+If Logstash is installed in your cluster, and you plan Epiphany upgrade from v1.x to v2.x, it is recommended to remove Logstash from your machines.
+
+**RedHat clusters**:
+1. Check if Logstash package is installed: `sudo yum info logstash-oss`
+2. Remove: `sudo yum remove logstash-oss`
+
+**Debian clusters**:
+1. Check if Logstash package is installed: `sudo apt --installed list | grep logstash-oss`
+2. Remove: `sudo apt-get remove logstash-oss`
```

Comment:
Should we describe the way of packages removal or just list them? For me these commands look obviously.

Reply:
I talked with @to-bar about packages removal, and like you can do it in at least 2 ways (the other ones are rpm and dpkg, according to the OS), they don't provide any logging for that action (e.g. yum history, or /var/log/apt/history.log). I don't know how much do we care about it, but I can either add comment about it in doc, or just part `If Logstash is installed in your cluster, and you plan Epiphany upgrade from v1.x to v2.x, it is recommended to remove Logstash from your machines.` What do you think?